exercises/practice/bowling/src/lib.rs

```diff
@@ -1,21 +1,223 @@
+const ROUNDS_NUMBER: usize = 10;
+const THROW_TYPES_AMOUNT: usize = 3;
+const PINS_AMOUNT: u16 = 10;
+
+const FIRST_THROW: usize = 0;
+const SECOND_THROW: usize = 1;
+const FILL_BALL_THROW: usize = 2;
+
 #[derive(Debug, PartialEq, Eq)]
 pub enum Error {
     NotEnoughPinsLeft,
     GameComplete,
 }
 
-pub struct BowlingGame {}
+#[derive(Debug, PartialEq, Eq)]
+enum ThrowType {
+    First,
+    Second,
+    FillBall,
+}
+
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+enum RoundResult {
+    Regular,
+    Spare,
+    Strike,
+}
+
+#[derive(Clone, Copy)]
+struct Frame {
```

Comment:
add doc,
put the struct and impl one after another preferably put different structs in different file